tests/test_flask_server.py

```diff
@@ -27,6 +27,17 @@ def client_fixture(manager):
     return app.test_client()
 
 
+def test_return_user_id_is_uuid(manager):
+    mock_job_manager = manager
+    app = create_app(mock_job_manager)
+    app.config['TESTING'] = True
+    client = app.test_client()
+    result = client.get('/get_user_id')
+    json_ = json.loads(result.data)
+    is_uuid = UUID(json_['user_id'], version=4)
```

Comment:
Does the constructor validate the value?  A comment about what is tested would help.